### PR TITLE
chore(sidebar): remove classic theme FE-3088

### PR DESCRIPTION
**Pull request by jamime**
### Proposed behaviour

```
BREAKING CHANGE: The classic theme has been removed from sidebar. To
upgrade please use a DLS theme.
```

- Classic sidebar stories removed
- Classic sidebar style removed

### Current behaviour

- Classic sidebar stories
- Classic sidebar style

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### Testing instructions
Ensure the stories have been removed.

### CODE REVIEW

**Review by edleeks87 — APPROVED** [review]

File: src/components/sidebar/sidebar.stories.js [hunk 1]
```diff
@@ -113,6 +113,4 @@ storiesOf('Sidebar', module)
     }
   })
   .add(...makeStory('default', dlsThemeSelector))
-  .add(...makeStory('classic', classicThemeSelector, true))
-  .add(...makeButtonStory('with button', dlsThemeSelector))
-  .add(...makeButtonStory('with button classic', classicThemeSelector, true));
+  .add(...makeButtonStory('with button', dlsThemeSelector));
```
Comment (grabkowski) [inline src/components/sidebar/sidebar.stories.js:116]: I think that we could move the `dlsThemeSelector` directly to the `metadata`, we no longer need to pass it as an argument

**Review by grabkowski — COMMENTED** [review]
shouldn't the `BREAKING CHANGE` be in a comment footer as per Conventional Commits spec?

File: src/components/sidebar/sidebar.stories.js (hunk 1 above)
Comment (jamime) [reply src/components/sidebar/sidebar.stories.js]: Resolved in 24df79c903de453e17595c2de56f7cabe2e3181d

**Comment by jamime** [comment]
> shouldn't the `BREAKING CHANGE` be in a comment footer as per Conventional Commits spec?

@grabkowski I think it is 

```
<type>[optional scope]: <description>

[optional body]

[optional footer(s)]
```

```
chore(sidebar): remove classic theme
    
BREAKING CHANGE: The classic theme has been removed from sidebar. To
upgrade please use a DLS theme.
    
Fixes FE-3088
```

So the first footer is (including the new line)
```
BREAKING CHANGE: The classic theme has been removed from sidebar. To
upgrade please use a DLS theme.
   
```

the second footer is
```
Fixes FE-3088
```

What do you think it should be changed to?

**Review by grabkowski — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Pull request merged**